specs/lib/getUniqueIdSpec.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+var getUniqueId = require('../../lib/getUniqueId');
+
+describe('getUniqueId', function() {
+    var gltf = {
+        "accessors": {
+            "accessor": {
+                "bufferView": "bufferView_137"
+            },
+            "accessor_0": {
+                "bufferView": "bufferView_137"
+            },
+            "accessor_1": {
+                "bufferView": "bufferView_138"
+            }
+        },
+        "buffers": {
+            "bufferId": {
+                "access": ''
+            }
+        },
+        "materials": {
+            "blinn-1": {},
+            "lambert-1": {}
+        }
+    };
+
+    var allIdentifiersGltf = {
+        "nodes": {
+            "name" : ''
```

Comment:
To make this slightly more true to the spec, replace `''` here and below with `{}`.

These glTF properties are objects, not strings.

Also, change `"name*"` to `"id*"` since these are ids, not names.  Some glTF properties also have a name (e.g., used for display, not internal references) so naming an id `"name"` can be confusing.